selfdrive/car/ford/values.py

```diff
@@ -213,6 +219,16 @@ def init_make(self, CP: car.CarParams):
       b'NB5A-14C204-HB\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00',
     ],
   },
+  CAR.F_150_LIGHTNING_MK1: {
+    (Ecu.shiftByWire, 0x732, None): [
+      b'ML3P-7P470-AK\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00',
+    ],
+  },
+  CAR.MUSTANG_MACH_E_MK1: {
+    (Ecu.shiftByWire, 0x732, None): [
+      b'ML3P-7P470-AK\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00',
+    ],
```

Comment:
Right now we require the OBD port to fingerprint, is the comma power plugged in?

Reply:
Yup. This is all that's currently returned for both the F150 Lightning and the Mustang Mach E. A Mach E user was able to drive on a branch for the F150 because it thinks it's a truck.

Reply:
Dongle ids?

Reply:
F150 Lightning: 112e4d6e0cad05e1

I'll post the Mach E ID when the user gets it to me.

Reply:
Mach E ID: 
42f08a29af3699f4